docker/versions.dockerfile

```diff
@@ -13,7 +13,7 @@ ARG HASKELL_STACK_VERSION=2.13.1
 ARG JAVA_VERSION=21
 ARG LICENSEE_VERSION=9.18.0
 ARG NODEJS_VERSION=22.13.0
-ARG NPM_VERSION=10.9.2
+ARG NPM_VERSION=11.0.0
```

Comment:
Didn't we say that we want this to match the version that comes with the latest Node LTS version? Also see 8ebd8c0aea0ffb0770edc92a8c477a4e2f4f0586.

Reply:
Yes, I figured out the latest NPM version for the currently used node (LTS) version by running `nvm install-latest-npm`, see also https://stackoverflow.com/questions/9755841/how-can-i-change-the-version-of-npm-using-nvm.

Reply:
Quote from your linked post about what that command does:

> Attempt to upgrade to the latest working npm on the current Node.js version.

So to my understanding this installs the latest (non-LTS) version of NPM that happens to work with the LTS version of Node.

But my understanding was that we'd want to stick to exactly that version of NPM that ships with the Node LTS version, and never update NPM independently of Node.

While I probably wouldn't have a problem changing that policy, I'd like to get a common understanding here.

Reply:
I see. I'm also fine with dropping the PR. What you wrote makes sense to me.

What do other think @oss-review-toolkit/core-devs  ?

Reply:
I think we should stick to the version that ships with Node LTS.